vizro-core/tests/e2e/vizro/conftest.py

```diff
@@ -1,35 +1,56 @@
+import os
+
 import e2e.vizro.constants as cnst
 import pytest
+import yaml
 from e2e.vizro.checkers import browser_console_warnings_checker
+from e2e.vizro.waiters import callbacks_finish_waiter
 from selenium.common import WebDriverException
 from selenium.webdriver.chrome.options import Options as ChromeOptions
 
 
 # dash_br_driver options hook
 def pytest_setup_options():
-    options = ChromeOptions()
-    options.add_argument("--headless")
-    options.add_argument("--window-size=1920,1080")
-    return options
+    if os.getenv("BROWSER") == "chrome_mobile":
+        options = ChromeOptions()
+        options.add_experimental_option("mobileEmulation", {"deviceName": "iPhone 14 Pro Max"})
+        return options
 
 
 def make_teardown(dash_br):
+    # rewriting dynamic_filters_data.yml after each test
+    data = {
+        "max": 7,
+        "min": 6,
+        "setosa": 5,
+        "versicolor": 10,
+        "virginica": 15,
+    }
+    with open(cnst.DYNAMIC_FILTERS_DATA_CONFIG, "w") as file:
+        yaml.dump(data, file)
     # checking for browser console errors
-    try:
-        log_levels = [level for level in dash_br.get_logs() if level["level"] == "SEVERE" or "WARNING"]
-        if log_levels:
-            for log_level in log_levels:
-                browser_console_warnings_checker(log_level, log_levels)
-    except WebDriverException:
-        pass
+    if os.getenv("BROWSER") == ("chrome" or "chrome_mobile"):
```

Comment:
In vizro-core code, the following way of comparison is mostly used.
```suggestion
    if os.getenv("BROWSER") in ["chrome", "chrome_mobile"]:
```

Reply:
Done